### PR TITLE
Posting comments

### PR DESCRIPTION
**Pull request by Bretl7**
Able to post comments to thread and adds these posts to DB

### CODE REVIEW

File: Shared/ForumDTO/Data.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+namespace SharpScape.Shared.Dto
+{
+    public class Data
```
Comment (cptchuckles) [inline Shared/ForumDTO/Data.cs:3]: Give this class a descriptive name

File: Website/Program.cs [hunk 2]
```diff
@@ -13,6 +13,8 @@
 builder.Services.AddScoped<IAuthTokenProvider, AuthTokenProvider>();
 builder.Services.AddScoped<AuthenticationStateProvider, AppAuthStateProvider>();
 builder.Services.AddScoped<IForumCategoryServices , ForumCategoryServices>();
+builder.Services.AddSingleton<SharpScape.Shared.Dto.Data>();
```
Comment (cptchuckles) [inline Website/Program.cs:16]: Define a service with a descriptive name that lives in `SharpScape.Website.Services`

**Review by cptchuckles — CHANGES_REQUESTED** [review]
`Data` is not a sufficiently descriptive name for a DTO or for a service.  Come up with a meaningful name for it that represents its *specific role* in the program.

DTOs are not services and shouldn't be used as such.  Define a service class in the `SharpScape.Website.Services` namespace to perform the role of a service.

File: Shared/ForumDTO/Data.cs (hunk 1 above)
Comment (Bretl7) [reply Shared/ForumDTO/Data.cs]: Changed name as requested

File: Website/Program.cs (hunk 2 above)
Comment (Bretl7) [reply Website/Program.cs]: Changed

**Comment by Bretl7** [comment]
Changed name to ForumReferenceProvider

**Pull request merged**